src/client/object/ObjectGallery/CreateAlbum.js

```diff
@@ -1,11 +1,60 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Button, Form, Input, Modal } from 'antd';
+import { Button, Form, Input, message, Modal } from 'antd';
+import { connect } from 'react-redux';
 import { FormattedMessage, injectIntl } from 'react-intl';
+
 import { objectNameValidationRegExp } from '../../../common/constants/validation';
+import { prepareAlbumData, prepareAlbumToStore } from '../../helpers/wObjectHelper';
+import { getAuthenticatedUserName, getIsAppendLoading, getObject } from '../../reducers';
+import { appendObject } from '../appendActions';
+import { addAlbumToStore } from './galleryActions';
 import './CreateAlbum.less';
 
-const CreateAlbum = ({ showModal, hideModal, handleSubmit, form, loading, intl }) => {
+const CreateAlbum = ({
+  showModal,
+  hideModal,
+  form,
+  loading,
+  intl,
+  currentUsername,
+  wObject,
+  // eslint-disable-next-line no-shadow
+  appendObject,
```

Comment:
better rename 'appendObject' props to 'dispatchAppendObject' or 'appendObjectDispatch' and remover eslint comment

Reply:
Done